sound/soc/sof/ipc.c

```diff
@@ -187,6 +187,13 @@ static inline void ipc_log_header(struct device *dev, u8 *text, u32 cmd)
 }
 #endif
 
+/* schedule work to transmit any IPC in queue */
+static void snd_sof_ipc_msgs_tx(struct snd_sof_dev *sdev)
```

Comment:
fix sof-priv.h then?

Reply:
also if this is a static maybe remove the snd_sof prefix?

Reply:
This function is completely removed in a later patch anyway. So in fact those two patches can be merged.